### DISCUSSION TITLE
Accessing SOC via Virtual IP (pfSense)

### DISCUSSION THREAD

**Discussion by Sazerak** [Security-Onion-Solutions/securityonion#9825] [Unsupported Versions] [answered]
I'm trying to access the SOC on a manager that is behind a firewall. I've assigned a Virtual IP and setup the relevant NAT rules in pfSense but I'm running into issues getting the browser to access SOC.

I've done a few similar setups with Virtual IPs, so I'm confident the issue isn't on that side. I can also SSH into the manager using the Virtual IP no problem. I've checked packets and things seem fine at first. The handshake completes and a certain amount of packets go back and forth using the Virtual IP, but then at some point, the packets start going to the real IP and obviously go nowhere.

Any idea what could be causing this? I noticed SO2 has an nginx container, but I'm not knowledgeable enough in this area to tell if it could be part of the problem (and I can't find any documentation showing the details of nginx in SO2).

**Comment by tomjones1977** [comment, marked as answer]
It's likely that your Nginx settings have the real IP of the manager as the URL.  When you connect your browser will get a redirect to the real IP, which will hang as it can no longer access it.  

The best setup would be to access it via a URL and have the correct DNS setup to point to the VIP.

A workaround (though not recommended) is to set the nginx settings to the VIP.

https://docs.securityonion.net/en/2.3/url-base.html

**Reply by Sazerak** [reply]
Thanks, I didn't think of that. Quick and easy solution.
